src/components/VisualTesterKeymap.vue

```diff
@@ -161,9 +187,22 @@ export default {
         evStr
       ].join(' ');
     },
+    getElapsedTime(ev, endTs) {
+      return (endTs - this.timing[ev.code]).toFixed(3);
+    },
+    enableCatchKeys() {
```

Comment:
I don't really like this naming. I think it should be something like enableInput and IgnoreInput

Reply:
refactored